kvmagent/kvmagent/test/localstorage_testsuite/test_file_metadata.py

```
@@ -0,0 +1,893 @@
+
+    def _test_validate_empty_string_raises(self):
+        with self.assertRaises(ValueError):
+            _validate_metadata_path('')
+
+    def _test_validate_no_suffix_raises(self):
+        with self.assertRaises(ValueError):
+            _validate_metadata_path('/tmp/somefile.json')
+
+    def _test_validate_valid_path_passes(self):
+        # should not raise
+        _validate_metadata_path('/tmp/aabbccdd' * 4 + '.vmmeta')
```

Ruff (0.15.7)

[error] Probable insecure usage of temporary file or directory: "/tmp/aabbccdd"

(S108)